index.js

```diff
@@ -2,8 +2,12 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
+const jsesc = require('jsesc');
+
 module.exports = function(content) {
 	this.cacheable && this.cacheable();
 	this.value = content;
-	return "module.exports = " + JSON.stringify(content);
+	content = JSON.stringify(content);
+	content = jsesc(content);
+	return 'module.exports = ' + content;
```

Comment:
If `jsesc` will be used we can go with a template literal here since the dependency requires `node >= v4.0.0` anyways. `node =< v4.0.0` is EOL, but this would be a SemVer Major for `json-loader` then.

Reply:
You could use jsesc v1.3.0: https://github.com/mathiasbynens/jsesc#support

Reply:
Well it depends, since `node =< v4.0.0` is EOL, it's more of a note and just needs consensus how to handle it (:label: `patch/major` &&` jsesc` v1.3.0/latest). Let's wait for input of others on this matter :)